coursedata/texts/nl.yaml

```diff
@@ -35,6 +35,7 @@ ClientErrorMessages:
     CheckInternet: "Controleer even of je internetverbinding het nog doet?"
     ServerError: "Je hebt een programma geschreven dat we niet verwacht hadden. Als je wilt helpen, stuur ons dan een mailtje met het level en je programma op hedy@felienne.com. Probeer om verder te gaan je programma een beetje aan te passen en kijk nog eens goed naar de voorbeelden. Bedankt!"
 HedyErrorMessages:
+    Empty Program: "Er was geen code gevonden. Typ graag wat code in."
```

Comment:
Wat een grappige formulering, je zou bijna denken dat het uit Translate komt. Ik fix zo wel even in ietsje indiomatischer.